cache-config/t3cutil/getdatacfg.go

```diff
@@ -638,29 +642,55 @@ func GetConfigData(toClient *toreq.TOClient, disableProxy bool, cacheHostName st
 		}
 		return nil
 	}
-	cacheKeyParamsF := func() error {
-		defer func(start time.Time) { log.Infof("cacheKeyParamsF took %v\n", time.Since(start)) }(time.Now())
+
+	cacheKeyConfigParamsF := func() error {
+		defer func(start time.Time) { log.Infof("cacheKeyConfigParamsF took %v\n", time.Since(start)) }(time.Now())
 		{
 			reqHdr := (http.Header)(nil)
 			if oldCfg != nil {
-				reqHdr = MakeReqHdr(oldCfg.MetaData.CacheKeyParams)
+				reqHdr = MakeReqHdr(oldCfg.MetaData.CacheKeyConfigParams)
 			}
-			params, reqInf, err := toClient.GetConfigFileParameters(atscfg.CacheKeyParameterConfigFile, reqHdr)
+			params, reqInf, err := toClient.GetConfigFileParameters("cachekey.config", reqHdr)
```

Comment:
Why change the constant to a literal?

Reply:
should one be added for remap.config and parent.config to make it uniform?

Reply:
That would be ideal, but IMO not important enough to hold up merging